assets/src/edit-story/components/form/advancedDropDown/dropDown.js

```diff
@@ -51,11 +56,18 @@ const DropDownSelect = styled.button`
   background-color: ${({ theme, lightMode }) =>
     lightMode
       ? rgba(theme.DEPRECATED_THEME.colors.fg.white, 0.1)
-      : rgba(theme.DEPRECATED_THEME.colors.bg.black, 0.3)};
+      : 'transparent'};
```

Comment:
Note: leaving in the deprecated lightMode etc. since that's still needed in the Document panel and needs to get addressed separately.